elements/rh-dialog/docs/10-style.md

```diff
@@ -26,7 +26,7 @@ A dialog is a floating container on top of a transparent backdrop. The container
 
 A dialog is available in the light theme only.
 
-![Light theme dialog]({{ '../dialog-theme-light.png' | url }})
+![Light theme dialog]({{ '../dialog-theme-light.png' | url }}){style="--inline-img-max-width:1000px;"}
```

Comment:
Like pagination, this should probably use example shortcode

Reply:
@bennypowers I was just following the design spec [here](https://docs.google.com/document/d/1lnv8plbMefcxMCM_kw0Cszc6xeJfIZ8OJ2lcOQ0n8mg/edit#heading=h.4evziz6xaql5), but happy to wrap these in our example container if @coreyvickery agrees.

Reply:
> @bennypowers I was just following the design spec [here](https://docs.google.com/document/d/1lnv8plbMefcxMCM_kw0Cszc6xeJfIZ8OJ2lcOQ0n8mg/edit#heading=h.4evziz6xaql5), but happy to wrap these in our example container if @coreyvickery agrees.

Disregard. Using the example shortcode with a palette of none.